content/contributing/style_guide.md

```diff
@@ -0,0 +1,19 @@
+Title: Style Guide
+Date: 2021-07-12
+Order: 4
+
+## Pull requests
+
+So you want to contribute code to BookWyrm: that rules! If there's an open issue that you'd like to fix, it's helpful to comment on the issue so work doesn't get duplicated. Try to keep the scope of pull requests small and focused on a single topic. That way it's easier to review, and if one part needs changes, it won't hold up the other parts.
+
+If you aren't sure how to fix something, or you aren't able to get around to it, that's totally okay, just leave a comment on the pull request and we'll figure it out 💖.
+
+Pull requests have to pass all the automated checks before they can be merged - this includes an accessibility linter, python style checks, global linters, a security check, and unit tests.
+
+## Code style
+
+BookWyrm uses the [Black](https://github.com/psf/black) code formatter to keep the Python codebase consistent styled. All new pull requests are checked with GitHub actions, and you can automatically fix code style problems by running `./bw-dev black`
+
+## Linting
```

Comment:
This is me being picky and no blocker but because black is technically a linter having it under code style as `### Linting` makes more sense to me. The information is really good and will help me make sure the code looks good.

Reply:
Thanks!